cmd/summary.go

```diff
@@ -25,6 +25,8 @@ var summaryCmd = &cobra.Command{
 
 func init() {
 	rootCmd.AddCommand(summaryCmd)
+
+	summaryCmd.Flags().StringVar(&summaryOptions.GRPC, "gRPC", "", "gRPC server information")
```

Comment:
Would "Specify gRPC port" be more verbose?

Reply:
@Ankurk99 I don't think is port alone. This will be **`IP:port`** combination. Hence server information LGTM.